blueman/gui/GtkAnimation.py

```diff
@@ -138,12 +142,14 @@ def on_draw(self, widget, cr):
         cr.set_operator(cairo.OPERATOR_OVER)
         cr.paint()
 
-    def state_changed(self, state):
+        return False
+
+    def state_changed(self, state: float) -> None:
         self.tw.queue_draw()
 
 
 class CellFade(AnimBase):
-    def __init__(self, tw, path, columns=None):
+    def __init__(self, tw: Gtk.TreeView, path: Gtk.TreePath, columns: Iterable[int]) -> None:
```

Comment:
tw is not a plain TreeView but ManagerDeviceList.

Reply:
Does it have to be? From the attributes `CellFade` expects I think any `Gtk.TreeView` should work.

Reply:
I think when the pygobject stubs are installed mypy will complain. But we can cross that bridge when that happens.